pkg/snapstore/snapstore_test.go

```diff
@@ -278,10 +280,281 @@ var _ = Describe("Save, List, Fetch, Delete from mock snapstore", func() {
 	})
 })
 
-func resetObjectMap() {
-	for k := range objectMap {
-		delete(objectMap, k)
+// directory where the credential files created are stored
+const (
+	credentialDirectory = "../../test/credential"
+)
+
+var (
+	// environment variables for each provider
+	providers = []string{
+		"AWS_APPLICATION_CREDENTIALS",
+		"AZURE_APPLICATION_CREDENTIALS",
+		"OPENSTACK_APPLICATION_CREDENTIALS", // V3ApplicationCredentials
+		"OPENSTACK_APPLICATION_CREDENTIALS", // Password
```

Comment:
what's meaning of same thing written twice ? IMO, you can use different name with `password` or `V3` as substring. WDYT ?